lib/matplotlib/image.py

```diff
@@ -1374,26 +1383,27 @@ def imsave(fname, arr, vmin=None, vmax=None, cmap=None, format=None,
     Parameters
     ----------
     fname : str or file-like
-        Path string to a filename, or a Python file-like object.
-        If *format* is *None* and *fname* is a string, the output
-        format is deduced from the extension of the filename.
+        The filename or a Python file-like object to store the image in.
     arr : array-like
-        An MxN (luminance), MxNx3 (RGB) or MxNx4 (RGBA) array.
-    vmin, vmax: [ None | scalar ]
+        The image data. The shape can be one of
+        MxN (luminance), MxNx3 (RGB) or MxNx4 (RGBA).
+    vmin, vmax : scalar, optional
         *vmin* and *vmax* set the color scaling for the image by fixing the
         values that map to the colormap color limits. If either *vmin*
         or *vmax* is None, that limit is determined from the *arr*
         min/max value.
-    cmap : matplotlib.colors.Colormap, optional
-        For example, ``cm.viridis``.  If ``None``, defaults to the
-        ``image.cmap`` rcParam.
-    format : str
-        One of the file extensions supported by the active backend.  Most
-        backends support png, pdf, ps, eps and svg.
-    origin : [ 'upper' | 'lower' ]
-        Indicates whether the ``(0, 0)`` index of the array is in the
-        upper left or lower left corner of the axes.  Defaults to the
-        ``image.origin`` rcParam.
+    cmap : str or `~matplotlib.colors.Colormap`, optional
+        A Colormap instance or registered colormap name. The colormap
+        maps scalar data to colors. It is ignored for RGB(A) data.
+        Defaults to :rc:`image.cmap`.
+    format : str, optional
+        The file format. One of the file extensions supported by the active
+        backend.  Most backends support png, pdf, ps, eps and svg.
+        If not given, the format is deduced form the filename extension in
+        *fname*.
+    origin : {'upper', 'lower'}, optional
+        Indicates whether the ``(0, 0)`` index of the array is in the upper
+        left or lower left corner of the axes.  Defaults to :rc:`image.origin`.
```

Comment:
I find these "Defaults to :rc:`blah.bloo`" not as helpful as they should be because I don't have the default rcparams.  I think we should parenthetically add the default rcParam value in these cases...